Documentation/installing-dependencies.rst

```diff
@@ -1,216 +1,293 @@
-A study comparing partition quality for SeisSol meshes can be found `here <https://home.in.tum.de/~schnelle/publications/bachelorsthesis-informatics-final.pdf>`_.
+Additionally, you need the following libraries:
 
+- MPI (Support for MPI Standard >= 2.2)
+- Eigen (>= 3.4)
+- hdf5 (>= 1.8)
+- easi (>= 1.2)
+- (optional) a code generator
 
-In addition, the following packages need to be installed for the GPU version of SeisSol:
+  - libxsmm (== 1.17, newer versions do not work with YATeTo right now)
+  - PSpaMM
+- (optional) mesh partitioning
 
-- CUDA (>= 11.0)  for Nvidia GPUs, or HIP (ROCm>= 5.2.0) for AMD GPUs
-- SYCL: either hipSYCL >= 0.9.3 or DPC++
-- gemmforge (>= 0.0.207)
-- chainforge (>= 0.0.3, for Nvidia and AMD GPUs)
+  - ParMETIS (needs METIS to be installed as well)
+  - ParHIP
+  - PT-SCOTCH
+- (optional) Netcdf (>= 4.4)
+- (optional) ASAGI
 
+All dependencies can be installed automatically with spack or manually by hand.
+For a minimal installation (e.g. for running SeisSol on your local machine),
+you may avoid all optional dependencies. However, for the maximal performance, and for compute clusters,
+we recommend having at least using a code generator and having a mesh partitioner linked to SeisSol.
 
-These dependencies can be installed automatically with spack or can be installed manually one by one.
+For the GPU version, the following packages need to be installed for the GPU version of SeisSol:
 
+- SYCL: either hipSYCL >= 0.9.3 or DPC++
+- gemmforge (>= 0.0.207)
+- (optional) chainforge (>= 0.0.2, for Nvidia and AMD GPUs)
+- (optional) CUDA (>= 11.0)  for Nvidia GPUs, or HIP (ROCm>= 5.2.0) for AMD GPUs
 
 .. _spack_installation:
 
 Spack installation
 ------------------
 
-`Spack <https://github.com/spack/spack/wiki>`_ is a HPC software package manager.
-It automates the process of installing, upgrading, configuring, and removing computer programs.
-In particular, our spack package `seissol-env` allows automatically installing all dependencies of SeisSol (e.g. mpi, hdf5, netcdf, easi, asagi, etc).
+The `Spack <https://github.com/spack/spack/wiki>`_ package `seissol-env` allows to automatically install all dependencies of SeisSol (e.g. mpi, hdf5, netcdf, easi, asagi, etc),
+leaving only SeisSol itself left to be build.
+
 See https://github.com/SeisSol/seissol-spack-aid/tree/main/spack for details on the installation with spack.
 See also for reference our documentation on how to compile seissol-env on :ref:`SuperMUC-NG <compile_run_supermuc>`, :ref:`Shaheen <compile_run_shaheen>` (Cray system) and :ref:`Frontera <compile_run_frontera>`.
 
-
-Manual installation
+Manual Installation
 -------------------
 
-Initial Adjustments to .bashrc
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend setting up a folder on your system which will contain the build files for all dependencies.
+For example, do `mkdir ~/seissol; cd seissol`.
```

Comment:
Do you really think `seissol` is a good name? It will clash with the name of the git repo. I'd use `~/seissol_dependencies`